src/style.css

```diff
@@ -1,3 +1,66 @@
-body {
+* {
```

Comment:
It's bad practice to reset some properties in `*` selector. Better list tags you really want to reset, for example:

```
body,
ul {
  margin: 0;
}
```
Directly in `*` selector you can set `box-sizing` property.

Just FYI

Reply:
i will remeber it in future . bit for now can you accept this reguest please? because i`m already tired of this task, and last times i wana to upload some changes it have some errors whith this branch